playground/CommandInfoUpdater/SupportedCommand.cs

```diff
@@ -0,0 +1,263 @@
+            new("SUBSCRIBE", RespCommand.SUBSCRIBE),
+            new("TIME", RespCommand.TIME),
+            new("TTL", RespCommand.TTL),
+            new("TYPE", RespCommand.TYPE),
+            new("UNLINK", RespCommand.UNLINK),
+            new("UNSUBSCRIBE", RespCommand.UNSUBSCRIBE),
+            new("UNWATCH", RespCommand.UNWATCH),
+            new("WATCH", RespCommand.WATCH),
+            new("WATCHMS", RespCommand.WATCHMS),
+            new("WATCHOS", RespCommand.WATCHOS),
+            new("ZADD", RespCommand.SortedSet, (byte) SortedSetOperation.ZADD),
+            new("ZCARD", RespCommand.SortedSet, (byte) SortedSetOperation.ZCARD),
+            new("ZCOUNT", RespCommand.SortedSet, (byte) SortedSetOperation.ZCOUNT),
+            new("ZDIFF", RespCommand.SortedSet, (byte) SortedSetOperation.ZDIFF),
+            new("ZINCRBY", RespCommand.SortedSet, (byte) SortedSetOperation.ZINCRBY),
+            new("ZLEXCOUNT", RespCommand.SortedSet, (byte) SortedSetOperation.ZLEXCOUNT),
+            new("ZMSCORE", RespCommand.SortedSet, (byte) SortedSetOperation.ZMSCORE),
+            new("ZPOPMAX", RespCommand.SortedSet, (byte) SortedSetOperation.ZPOPMAX),
+            new("ZPOPMIN", RespCommand.SortedSet, (byte) SortedSetOperation.ZPOPMIN),
+            new("ZRANDMEMBER", RespCommand.SortedSet, (byte) SortedSetOperation.ZRANDMEMBER),
+            new("ZRANGE", RespCommand.SortedSet, (byte) SortedSetOperation.ZRANGE),
+            new("ZRANGEBYSCORE", RespCommand.SortedSet, (byte) SortedSetOperation.ZRANGEBYSCORE),
+            new("ZRANK", RespCommand.SortedSet, (byte) SortedSetOperation.ZRANK),
+            new("ZREM", RespCommand.SortedSet, (byte) SortedSetOperation.ZREM),
+            new("ZREMRANGEBYLEX", RespCommand.SortedSet, (byte) SortedSetOperation.ZREMRANGEBYLEX),
+            new("ZREMRANGEBYRANK", RespCommand.SortedSet, (byte) SortedSetOperation.ZREMRANGEBYRANK),
+            new("ZREMRANGEBYSCORE", RespCommand.SortedSet, (byte) SortedSetOperation.ZREMRANGEBYSCORE),
+            new("ZREVRANGE", RespCommand.SortedSet, (byte) SortedSetOperation.ZREVRANGE),
+            new("ZREVRANK", RespCommand.SortedSet, (byte) SortedSetOperation.ZREVRANK),
+            new("ZSCAN", RespCommand.SortedSet, (byte) SortedSetOperation.ZSCAN),
+            new("ZSCORE", RespCommand.SortedSet, (byte) SortedSetOperation.ZSCORE),
+        };
+
+        /// <summary>
+        /// Supported command's name
+        /// </summary>
+        public string Command { get; set; }
+
+        /// <summary>
+        /// Supported command's sub-commands' names
+        /// </summary>
+        public HashSet<string>? SubCommands { get; set; }
+
+        /// <summary>
+        /// Garnet RespCommand
+        /// </summary>
+        public RespCommand RespCommand { get; set; }
+
+        /// <summary>
+        /// Garnet ArrayCommand
+        /// </summary>
+        public byte? ArrayCommand { get; set; }
+
+        public SupportedCommand()
```

Comment:
I did :) Since this class is being serialized, it needs to have an explicit parameterless constructor (since it also has a non-parameterless constructor)

Reply:
That makes sense. I would personally add that explicitly as a comment. Something like this:

```suggestion
        /// <summary>
        /// Default constructor provided for JSON serialization
        /// </summary>
        public SupportedCommand()
```